src/librustdoc/html/render.rs

```diff
@@ -1888,11 +1888,11 @@ fn short_stability(item: &clean::Item, cx: &Context, show_reason: bool) -> Vec<S
                 if stab.unstable_reason.is_empty() {
                     stability.push(format!("<div class='stab unstable'>\
                                             <span class=microscope>🔬</span> \
-                                            This is a nightly-only experimental API. {}</div>",
```

Comment:
There's no difference between a regular space and an `&nbsp`, nbsp just lets you string together two spaces.